python_modules/libraries/dagster-components/dagster_components/components/shim_components/schedule.py

```diff
@@ -0,0 +1,24 @@
+from pathlib import Path
+
+from dagster._utils import pushd
+
+from dagster_components import Scaffolder, ScaffoldRequest
+from dagster_components.components.shim_components.base import ShimComponent
+from dagster_components.scaffoldable.decorator import scaffoldable
+
+
+class ScheduleScaffolder(Scaffolder):
+    def scaffold(self, request: ScaffoldRequest, params: None) -> None:
+        with pushd(str(request.target_path)):
+            Path("definitions.py").write_text("""
+# import dagster as dg
+# 
+# @dg.schedule(cron_schedule=..., target=...)
+# def my_schedule(): ...
```

Comment:
can we scaffold an accurate typehint?

Reply:
The return values accepted by `@schedule`-decorated functions are [a mess](https://sourcegraph.com/github.com/dagster-io/dagster/-/blob/python_modules/dagster/dagster/_core/definitions/schedule_definition.py?L58-60) so we keep that blank in all of our docs